client/my-sites/ads/main.jsx

```diff
@@ -88,12 +91,18 @@ const AdsMain = React.createClass( {
 
 	renderInstantActivationToggle: function( component ) {
 		return ( <div>
+			<QueryWordadsStatus siteId={ this.props.site.ID } />
```

Comment:
Won't even me rendered by the controller:
https://github.com/Automattic/wp-calypso/blob/master/client/lib/ads/utils.js#L19